src/renderer/components/Updater/Banner.js

```diff
@@ -51,12 +53,18 @@ const CONTENT_BY_STATUS = (quitAndInstall, reDownload, progress): { [string]: Co
 
 const UpdaterTopBanner = () => {
   const context = useContext(UpdaterContext);
+  const remoteConfig = useRemoteConfig();
 
   const reDownload = useCallback(() => {
     openURL(urls.liveHome);
   }, []);
 
-  if (context) {
+  if (context && remoteConfig && context.version) {
```

Comment:
⚠️  this makes the release not resilient to network down problem or problem in the fetching config.

I think by default we always should suggest the auto update when fetching config fails. the challenge might be to not having it "flash" during the loading phase, so you might want to track if it's loading/error/loaded.

Reply:
I fixed it by adding a "lastUpdatedAt" param that ensure that we tried at least once to fetch the remote config, sucessfully or not.